scripts/configs.js

```diff
@@ -87,13 +92,21 @@ export const getConfigValue = async (configParam, environment) => {
 export const getHeaders = async (scope, environment) => {
   const env = environment || calcEnvironment();
   const config = await getConfigForEnvironment(env);
-  const configElements = config.data.filter((el) => el?.key.includes(`headers.${scope}`));
+  const configElements = config.data.filter((el) => el?.key.includes('headers.all') || el?.key.includes(`headers.${scope}`));
 
   return configElements.reduce((obj, item) => {
     let { key } = item;
+
+    // global values
+    if (key.includes('commerce.headers.all.')) {
```

Comment:
The comment on the fallback logic applies here as well.

Reply:
How so? We don't have a mechanism to set headers that apply to all "Core" requests. The idea behind `commerce.headers.all` is to provide headers that apply to both Core, CS, and all new services endpoints.

Reply:
Ah okay, nevermind then. I thought it was global headers for all languages :)